src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -564,8 +564,6 @@ public override SchemaShape GetOutputSchema(SchemaShape inputSchema)
                 var input = Transformer.Inputs[i];
                 if (!inputSchema.TryFindColumn(input, out var col))
                     throw Host.ExceptSchemaMismatch(nameof(inputSchema), "input", input);
-                if (!(col.Kind == SchemaShape.Column.VectorKind.VariableVector || col.Kind == SchemaShape.Column.VectorKind.Vector))
-                    throw Host.ExceptSchemaMismatch(nameof(inputSchema), "input", input, "vector", col.GetTypeString());
 
```

Comment:
if input col.Kind == VariableVector, how is that case handled? If not handled, should throw exception.

Reply:
so prior to this change, we allowed VariableVector in that it would not throw -- this change now allows for vector and simple types. I would prefer to have a test that uses  VariableVector, but I have yet to find something that provides a VariableVector output AND can be saved to ONNX.

Reply:
@jignparm from our conversation, you mentioned that we should not have allowed VariableVector to begin with and should fix as part of this change. I have updated the code to reflect that, but we should have a test that confirms if this works correctly. I have created the following issue for tracking #3375